.circleci/run.sh

```diff
@@ -109,7 +109,7 @@ run_make()
     make -v
 
     # -j1 is used for a better error log
-    make -f posix.mak -j1 RELEASE=1 DIFFABLE=1 release
+    make -f posix.mak -j1 DIFFABLE=1 release
```

Comment:
Shouldn't we set `ENABLE_RELEASE=1` here if we're about to renable this?

edit: saw the Makefile re-invocation below. Fair enough then.

Reply:
Yes, wanted to test that re-invocation ;).